routes/api/category-routes.js

```diff
@@ -1,28 +1,58 @@
-const router = require('express').Router();
-const { Category, Product } = require('../../models');
+const router = require("express").Router();
+const { response } = require("express");
+const { Category, Product } = require("../../models");
 
 // The `/api/categories` endpoint
 
-router.get('/', (req, res) => {
+router.get("/", async (req, res) => {
   // find all categories
+  const allCategories = await Category.findAll();
+  res.send(allCategories);
```

Comment:
consider responding also with a status - these are widely used in api responses throughout the internet e.g.
```
return res.status(200).json(allCategories)
```

Here's a breakdown of the main status codes and their meanings:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status